Sample Applications/ExpenseIt/ExpenseItDemo/ExpenseItDemo.csproj

```diff
@@ -1,111 +1,21 @@
 <Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
+
   <PropertyGroup>
     <TargetFramework>netcoreapp3.0</TargetFramework>
-    <UseWpf>true</UseWpf>
-    <EnableDefaultItems>false</EnableDefaultItems>
-  </PropertyGroup>
-  <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProjectGuid>{D1729F17-99A5-45AF-AB38-72FA6ECCE609}</ProjectGuid>
-    <OutputType>WinExe</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>ExpenseItDemo</RootNamespace>
-    <AssemblyName>ExpenseItDemo</AssemblyName>
-    <FileAlignment>512</FileAlignment>
-    <WarningLevel>4</WarningLevel>
-    <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
-    <PlatformTarget>AnyCPU</PlatformTarget>
-    <DebugSymbols>true</DebugSymbols>
-    <DebugType>full</DebugType>
-    <Optimize>false</Optimize>
-    <OutputPath>bin\Debug\</OutputPath>
-    <DefineConstants>DEBUG;TRACE</DefineConstants>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
+    <UseWPF>true</UseWPF>
+	  <OutputType>WinExe</OutputType>
+    <RootNamespace>ExpenseIt9</RootNamespace>
+    <AssemblyName>ExpenseIt9</AssemblyName>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
```

Comment:
Why do we need to turn this off?

Reply:
Same reason as [CustomComboBox review](https://github.com/microsoft/WPF-Samples/pull/248)